analytics_queries/participants.sql

```diff
@@ -12,6 +12,7 @@ SELECT u.id                                                    as participant_id
        c.start_year                                            as cohort,
        pp.training_status                                      as training_status,
        s.name                                                  as schedule,
+       epvd.trn                                                as trn,
```

Comment:
We should use the TRN on the teacher profile instead of this one. This one won't be retained forever, and also if they made a mistake in entering it that we could automatically correct with the other data then the teacher profile will be correct and this will be as-entered